README.md

```diff
@@ -144,7 +144,7 @@ Be aware that Push Notifications are currently unsupported for macOS and tvOS. Y
 
 ```swift
 let recipient: [String: Any] = [
-    "clientId": "C04BC116-8004-4D78-A71F-8CA3122734DB"
+    "clientId": "C04BC116-8004-4D78-A71F-8CA31.2.44DB"
```

Comment:
Oops. I'll fix that. 😬

Reply:
Difference between `1.2.3` and `1\.2\.3` when `sed` is using regex. 😁 
See `Scripts/set-version.sh`.

Reply:
Reverted: https://github.com/ably/ably-cocoa/pull/1114/commits/bc62b6cb9d2ae8362d545d14455037850b3af686.